src/dss/main.py

```diff
@@ -30,5 +31,43 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+@main.command(name="logs")
+@click.option(
+    "--kubeconfig",
+    help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
+)
+@click.argument("notebook_name", required=False)
+@click.option(
+    "--all", "print_all", is_flag=True, help="Print the logs for all notebooks and MLflow."
+)  # noqa: E501
+@click.option("--mlflow", is_flag=True, help="Print the logs for the MLflow deployment.")
+def logs_command(kubeconfig: str, notebook_name: str, print_all: bool, mlflow: bool) -> None:
+    """
+    Prints the logs for the specified notebook or DSS component.
+    """
+    if not notebook_name and not mlflow and not print_all:
+        click.echo(
+            "Failed to retrieve logs. Missing notebook name. Run the logs command with desired notebook name."  # noqa E501
+        )
+        return
+
+    kubeconfig = get_default_kubeconfig(kubeconfig)
+    lightkube_client = get_lightkube_client(kubeconfig)
+
+    if print_all:
+        get_logs("all", None, lightkube_client)
+    elif mlflow:
+        get_logs("mlflow", None, lightkube_client)
+    elif notebook_name:
+        get_logs("notebooks", notebook_name, lightkube_client)
+
+
+# Define Examples section for logs command
+logs_command.help += """
```

Comment:
Is this how a coomand `help` field should be defined? I mean outside its `def`inition? It's more of a question since it feels a bit weird but I 'm not a Python expert either.

Reply:
What about putting in the doc string like described here? https://click.palletsprojects.com/en/8.1.x/documentation/#preventing-rewrapping

Reply:
That helped I refactored the help message. Thanks.